### PR TITLE
Question page

### PR DESCRIPTION
**Pull request by Mohitkumar33**
https://user-images.githubusercontent.com/46507817/153882799-c46872e9-4123-450d-80d5-1c5bd147ca03.mp4

Here is the Question Page and the result page. If you find any mistake and if you think anything not looking good then please add a review. Thanks

### CODE REVIEW

**Review by ajay-mandaviya — APPROVED** [review]
everything looks clean ,

File: questions/question.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    justify-content: space-between;
+}
+.timmer {
+    position: fixed;
+    top: 0;
+    right: 1rem;
+}
+
+/* .button{
+    display: flex;
```
Comment (Prabhav-Sharma) [inline questions/question.css]: Delete unuseful commented code

File: questions/question.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    flex-direction: column;
+    justify-content: center;
+} */
+.button {
+    margin-top: 2.5rem;
```
Comment (Prabhav-Sharma) [inline questions/question.css:44]: Can you global variables for properties like margin, padding and gap

File: questions/question.css (hunk 1 above)
Comment (Mohitkumar33) [reply questions/question.css]: Thanks for the review Prabhav

File: questions/question.css (hunk 2 above)
Comment (Mohitkumar33) [reply questions/question.css:44]: Yes Prabhav I will do that

**Pull request merged**